packages/venia-concept/src/ServiceWorker/Utilities/messageHandler.js

```diff
@@ -9,6 +9,11 @@
  */
 const handlers = {};
 
+/**
+ * Exporting for testing purposes
+ */
+export const __handlers__ = handlers;
```

Comment:
Good comment. I'd love to be able to drop this, per other feedbacks.

Reply:
Had a chat with @zetlen a couple weeks ago where he was talking about modifying function behavior to make testing possible. Took his advice and made this change but I needed to let future developers know that `__handlers__` is strictly for testing only. But yeah I am up for suggestions.